pkg/apis/storage/validation/validation_test.go

```diff
@@ -47,6 +47,17 @@ func TestValidateStorageClass(t *testing.T) {
 				"foo-parameter2":              "{\"embedded\": \"json\", \"with\": {\"structures\":\"inside\"}}",
 			},
 		},
+		{
+			// empty reclaimPolicy
+			ObjectMeta:  metav1.ObjectMeta{Name: "foo"},
+			Provisioner: "kubernetes.io/foo-provisioner",
```

Comment:
we should add a test for delete reclaim as well, just to make sure explicitly specifying works as well.